firebase/functions/src/index.ts

```diff
@@ -11,7 +9,7 @@ export { bucketChange } from './uploads/add-outputs';
 export { postUserCreate } from './post-user-create';
 export { postHandshakeCommitCreate } from './post-handshake-write';
 
-const conf = functions.config().firebase;
+const conf = JSON.parse(process.env.FIREBASE_CONFIG);
 
 const serviceAccount = functions.config().fb_service_account;
```

Comment:
I thought `functions.config()` was gone entirely (hence the line above). Or is it still there?

Reply:
Can you elaborate on the config migration? Do we manually set the `FIREBASE_CONFIG` environment var or is this a built-in available env that exposes the exact same config that was previously available at `functions.config().firebase`?

Reply:
Only `firebase.config().firebase` is gone.

`FIREBASE_CONFIG` is provided by process, so we don't have to worry about it. It's basically the new way to retrieve your firebase config inside functions environment.